jruby/plan.sh

```diff
@@ -1,11 +1,11 @@
 pkg_origin=core
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_name=jruby
-pkg_version=9.1.2.0
+pkg_version=9.2.0.0
 pkg_description="A high performance, stable, fully threaded Java implementation of the Ruby programming language."
 pkg_upstream_url=https://github.com/jruby/jruby
 pkg_source=https://github.com/jruby/jruby/archive/${pkg_version}.tar.gz
-pkg_shasum=869434d400ea86dfa99622a83c662d6ac244c2a761928ce10f48d774f4fdeb0e
+pkg_shasum=3d59bde1639c69965664ee46a07be230141ee8e99d2e7f43b574a6a8298c887c
 pkg_license=('EPL 1.0, GPL 2 and LGPL 2.1')
 pkg_deps=(core/glibc core/jre8 core/bash)
```

Comment:
Could you move `core/coreutils` from `pkg_build_deps` to `pkg_deps`?  

When running `hab pkg exec core/jruby irb` , the following error occurs (present in existing packages as well)

```
/hab/pkgs/core/jruby/9.1.2.0/20180609043402/bin/jruby: line 17: uname: command not found
/hab/pkgs/core/jruby/9.1.2.0/20180609043402/bin/jruby: line 29: dirname: command not found
/hab/pkgs/core/jruby/9.1.2.0/20180609043402/bin/jruby: line 29: cd: : No such file or directory
/hab/pkgs/core/jruby/9.1.2.0/20180609043402/bin/jruby: line 44: dirname: command not found
/hab/pkgs/core/jruby/9.1.2.0/20180609043402/bin/jruby: line 49: dirname: command not found
Error: Could not find or load main class org.jruby.Main
```